pkg/apis/forklift/v1beta1/plan.go

```diff
@@ -44,6 +44,11 @@ type PlanSpec struct {
 	TransferNetwork *core.ObjectReference `json:"transferNetwork,omitempty"`
 	// Whether this plan should be archived.
 	Archived bool `json:"archived,omitempty"`
+	// LUKs keys for virt-v2v (https://www.libguestfs.org/virt-v2v.1.html)
+	// References a secret containing credentials and
+	// other confidential information.
+	// +optional
+	Secret core.ObjectReference `json:"secret" ref:"Secret"`
```

Comment:
`Secret` is a very generic name :) how about LUKS or EncryptionKeys?

Reply:
not sure if you meant only for the name we use in our code or also on the CRD. i changed only the name we use in the code.

Reply:
isn't it generated (the CRD)?

Reply:
yes, but i changed only `Secret`. to change the CRD I need:
```LUKS core.ObjectReference `json:"luks" ref:"Secret"```